app/assets/javascripts/components/story/StoryItem.js

```diff
@@ -53,7 +55,7 @@ StateActions.defaultProp = {
 };
 
 export const StateButton = ({ action }) => (
-  <button type="button" className={`Story__btn Story__btn--${action}`}>{action}</button>
+  <button type="button" className={`Story__btn Story__btn--${action}`}>{ I18n.translate('story.events.' + action) }</button>
```

Comment:
What do you think about breaking this line for better readability?
```suggestion
  <button type="button" className={`Story__btn Story__btn--${action}`}>
    { I18n.translate('story.events.' + action) }
  </button>
```